### ISSUE TITLE
Error with v11.0.5 (retail)

### ISSUE THREAD

**Issue by Gaviin1242** [Jaliborc/PetTracker#404]
Getting the following error during pet battle WQ in Isle of Dorn.

1x .../PetTracker/addons/main/classes/abstract/ability.lua:17: Usage: GetAttackModifier(attackType, defenseType)
[string "=[C]"]: in function `GetAttackModifier'
[string "@PetTracker/addons/main/classes/abstract/ability.lua"]:17: in function `GetModifier'
[string "@PetTracker/addons/main/classes/combat/abilityButton.lua"]:27: in function `Display'
[string "@PetTracker/addons/main/classes/combat/petSlot.lua"]:72: in function `Display'
[string "@PetTracker/addons/battle/switcher.lua"]:78: in function `UpdateFor'
[string "@PetTracker/addons/battle/switcher.lua"]:68: in function `Update'
[string "@PetTracker/addons/battle/switcher.lua"]:28: in function `PetBattlePetSelectionFrame_Show'
[string "@Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua"]:205: in function `PetBattleFrame_UpdatePetSelectionFrame'
[string "@Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua"]:119: in function <Blizzard_PetBattleUI/Blizzard_PetBattleUI.lua:108>

Locals:
(*temporary) = nil
(*temporary) = 10

**Comment by AlexL1118** [comment]
Yes, it looks like `self:GetInfo()` sometimes returns `type` as `nil`. I did not understand when and why.
